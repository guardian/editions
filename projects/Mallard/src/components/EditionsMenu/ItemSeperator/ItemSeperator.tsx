import React from 'react'
import { View } from 'react-native'
import { color } from 'src/theme/color'

const ItemSeperator = () => (
    <View
        style={{
            backgroundColor: color.palette.neutral[7],
            height: 1,
        }}
    ></View>
)

export { ItemSeperator }
