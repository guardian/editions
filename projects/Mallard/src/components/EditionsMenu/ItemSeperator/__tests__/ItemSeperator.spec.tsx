import React from 'react';
import TestRenderer from 'react-test-renderer';
import { ItemSeperator } from '../ItemSeperator';

describe('ItemSeperator', () => {
	it('should display a default ItemSeperator with correct styling used for RegionButton seperation', () => {
		const component = TestRenderer.create(<ItemSeperator />).toJSON();
		expect(component).toMatchSnapshot();
	});
});
