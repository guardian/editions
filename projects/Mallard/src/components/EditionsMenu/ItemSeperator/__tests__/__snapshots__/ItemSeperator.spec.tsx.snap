// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemSeperator should display a default ItemSeperator with correct styling used for RegionButton seperation 1`] = `
<View
  style={
    {
      "height": 10,
    }
  }
/>
`;
